package/whiteboard-hypercard/package

```diff
@@ -1,32 +1,34 @@
 #!/usr/bin/env bash
-# Copyright (c) 2021 The Toltec Contributors
+# Copyright (c) 2023 The Toltec Contributors
 # SPDX-License-Identifier: MIT
 
 pkgnames=(whiteboard-hypercard)
 pkgdesc="Real-time collaboration, drawing or whiteboarding"
 url=https://github.com/fenollp/reMarkable-tools
-pkgver=0.3.4-1
-timestamp=2021-10-25T15:45Z
+pkgver=0.3.7-1
+timestamp=2023-10-11T12:57Z
 section="drawing"
 maintainer="Pierre Fenoll <pierrefenoll@gmail.com>"
-license=Apache-2.0
+license=CC-BY-NC-ND
```

Comment:
I didn't realize the code had been relicensed? This is up to you of course, but I would recommend looking closely at the impacts of creative commons on code in all jurisdictions. I seem to recall there being some impacts in germany or something.